src/command/cmds/index.ts

```diff
@@ -1,21 +1,29 @@
-import type { Class, NonEmptyObject } from "type-fest";
-import { Ping } from "./Ping";
+import { readdirSync } from "fs";
+import { join } from "path";
+import type { Class } from "type-fest";
 import type { CommandOptions } from "../../types/commands";
-import { Find } from "./Find";
-import { Sb } from "./Sb";
-import { Help } from "./Help";
-import { ClearWorld } from "./ClearWorld";
+
+// Dynamic command loading
+const commandFiles = readdirSync(__dirname).filter((file) => file.endsWith(".ts") && file !== "index.ts");
```

Comment:
I would recommend using async import instead, since typescript standard

Reply:
const commandFiles = readdirSync(__dirname).filter((file) => file.endsWith(".ts") && file !== "index.ts");

on this line ?

Reply:
> const commandFiles = readdirSync(__dirname).filter((file) => file.endsWith(".ts") && file !== "index.ts");
> 
> on this line ? could you explain to me, it runs normally but i dont know where to apply your suggestions..

i think what he meant is replace the ```readdirSync``` with the async equivalent, since readdirSync is synchronous and is blocking.

Reply:
Hey, thanks mate. ill attach that on the next update. But somehow the current source on main is throwing an error when you build and run it. 

here is the error message, hoping anyone reading this know how to fix it 

`C:\Users\ASUS\Documents\projects\GrowServer\dist\chunk-AKDZNQZ3.js:19
var DoorTile = (_class = class extends Tile {
               ^

TypeError: Class extends value undefined is not a constructor or null      
    at Object.<anonymous> (C:\Users\ASUS\Documents\projects\GrowServer\dist\chunk-AKDZNQZ3.js:19:16)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Object..js (node:internal/modules/cjs/loader:1689:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32)
    at Function._load (node:internal/modules/cjs/loader:1128:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:141:16)
    at Object.<anonymous> (C:\Users\ASUS\Documents\projects\GrowServer\dist\chunk-6EDHOYDE.js:3:24)`

Reply:
Sorry for late answer, I see the problem there
Since the source using tsup as compiler it result error above.

I might consider changing the compiler into tsc instead.

Reply:
Let me fix the rest

Reply:
I'll co author this PR instead of closing it

Reply:
thank you mate, that was fast. i will learn from this mistakes. 😄